ui/src/admin/components/chronograf/AdminTabs.js

```diff
@@ -71,6 +71,13 @@ const AdminTabs = ({
 
 const {arrayOf, bool, func, shape, string} = PropTypes
 
+AdminTabs.defaultProps = {
+  organization: {
+    name: '',
```

Comment:
This could be `name: 'Default'`. I believe the org with id 0 is always Default

Reply:
It's not always Default. The Default org can have its name changed.